netmap/netmap_contract.go

```diff
@@ -601,6 +602,23 @@ func ListConfig() []record {
 	return config
 }
 
+// RegisterNewEpochRecipient registers passed contract as a NewEpoch event
+// receiver. Such a contract must have a `NewEpoch` method with a single numeric
+// argument. Transactions that call RegisterNewEpochRecipient must be witnessed
+// by the Alphabet.
+// Produces `NewEpochHookRegistration` notification event with a just registered
+// recipient in a success case.
+func RegisterNewEpochRecipient(contract interop.Hash160) {
```

Comment:
Do you need to pass `contract`? Likely you can take caller script hash.

Reply:
> Likely you can take caller script hash

yes, that is what `common` pkg does in this PR. it is a kinda of feature that allows you to register some contracts to test or fix smth manually

Reply:
> Likely you can take caller script hash

yes, that is what `common` pkg does in this PR. it is a kinda of feature that allows you to register some contracts to test or fix smth manually